docs/src/modules/utils/useRouterExtra.ts

```diff
@@ -0,0 +1,22 @@
+import { useRouter } from 'next/router';
+
+export default function useRouterExtra() {
+  const router = useRouter();
+  const asPathWithoutLang = router.asPath.replace(/^\/[a-zA-Z]{2}\//, '/');
+  let product = asPathWithoutLang.replace(/^\/([^/]+)\/.*/, '$1');
+  if (asPathWithoutLang.startsWith('/x')) {
+    product = asPathWithoutLang.replace('/x/react-', '').replace(/\/.*/, '');
+  }
+  return {
+    ...router,
+    asPathWithoutLang,
+    product: product as 'base' | 'material-ui' | 'joy-ui' | 'system' | 'data-grid' | 'date-pickers',
+    isMuiBase: product === 'base',
+    isMaterialUI: product === 'material-ui',
+    isJoyUI: product === 'joy-ui',
+    isMuiSystem: product === 'system',
+    isMuiX: asPathWithoutLang.startsWith('/x'),
+    isDataGrid: product === 'data-grid',
+    isDatePickers: product === 'date-pickers',
```

Comment:
data grid and date picker are not products, so I would expect:

```suggestion
    isMuiBase: product === 'base',
    isMaterialUI: product === 'material-ui',
    isJoyUI: product === 'joy-ui',
    isMuiSystem: product === 'system',
    isMuiX: asPathWithoutLang.startsWith('/x'),
```

but honestly, we have `product` so, instead of using `===` checks and

```suggestion
```

would likely be clearer.

Reply:
Do I only erase this section?! Not sure which suggestion to follow.

Reply:
I have removed them.